sample_apps_reinvent2021/go/scheduled-query-sample.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/aws/aws-sdk-go/service/timestreamquery"
+	"go_sample/utils"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/aws/aws-sdk-go/service/timestreamwrite"
+)
+
+func main() {
+
+	databaseName := utils.DATABASE_NAME
+	tableName := utils.TABLE_NAME
+	sqSampleAppTopicName := "sq_sample_app_topic.fifo"
+	sqSampleAppQueueName := "sq_sample_app_queue.fifo"
+	sqResultsDatabaseName := utils.SQ_DATABASE_NAME
+	sqResultsTableName := utils.SQ_TABLE_NAME
+	region := flag.String("region", "us-east-1", "region")
+	csvFilePath := flag.String("csv_file_path", "../data/sample.csv", "sample csv file path to"+
+		" ingest records into the table used for scheduled query")
+	deleteResourcesFlag := flag.Bool("delete_resources_after_execution_flag", true, "true/false to"+
+		" delete created resources after execution of this code")
+	runInvalidScheduledQueryFlag := flag.Bool("run_invalid_scheduled_query_flag", false, "false/true to"+
+		" run valid/invalid scheduledQuery E2E flow")
+	flag.Parse()
+	ok, err := utils.FileExists(*csvFilePath)
+	if !ok {
+		if errors.Is(err, os.ErrNotExist) {
+			utils.HandleError(err, fmt.Sprintf("csv file path provided=%s doesn't exist. Please "+
+				"provide with a valid csv file path\n", *csvFilePath), true)
+		}
+		utils.HandleError(err, fmt.Sprintf("csv file path provided=%s is invalid. Please "+
+			"provide with a valid csv file path\n", *csvFilePath), true)
+	}
+
+	sess, err := utils.GetSession(*region)
+	utils.HandleError(err, "Failed to start a new session", true)
+
+	utils.HandleError(err, "Failed to start a new session", true)
+
+	writeSvc := timestreamwrite.New(sess)
+	querySvc := timestreamquery.New(sess)
+
+	snsSvc := sns.New(sess)
```

Comment:
what about different regions?

Reply:
we already do `sess, err := utils.GetSession(*region)` on L48 I believe.  Am I missing something here?